src/app/builder/builder.component.html

```diff
@@ -107,9 +111,16 @@
         </fieldset>
 
         <fieldset>
-          <img [src]="props.imageUrl || defaults.imageUrl" id="artwork-preview">
-          <label>Artwork URL</label>
-          <input type="text" [(ngModel)]="props.imageUrl" id="imageUrl" [placeholder]="defaults.imageUrl" name="imageUrl">
+          <img [src]="props.epImageUrl || defaults.epImageUrl" id="artwork-preview">
+          <label>Episode Artwork URL</label>
+          <p><small>This artwork will be featured prominently in the foreground of the player.</small></p>
+          <input type="text" [(ngModel)]="props.epImageUrl" id="epImageUrl" [placeholder]="defaults.epImageUrl" name="epImageUrl">
```

Comment:
This is probably just dumb on my part, but I kind of thought this field was disabled and was confused about how to use it because I couldn't select the placeholder text. Maybe it could just be the default value instead of the placeholder text?